assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm

```diff
@@ -120,8 +120,8 @@ function printView(url) {
 							<a href="#toolLinkParam("$action" "doPrep_download_all" "view=$!view")" class="button" id="downloadAll" title="$!tlang.getString('downall')">$!tlang.getString('downall')</a>
 						#if(!$disableGrade)
 							## upload all
-							<a href="#toolLinkParam("$action" "doPrep_upload_all" "view=$view")" class="button" id="uploadAll" title="$!tlang.getString('uploadall.title')">$!tlang.getString('uploadall.title')</a>
-							<a href="#"
+							| <a href="#toolLinkParam("$action" "doPrep_upload_all" "view=$view")" class="button" id="uploadAll" title="$!tlang.getString('uploadall.title')">$!tlang.getString('uploadall.title')</a>
+							| <a href="#"
```

Comment:
Screen readers will read this as "vertical pipe" in between each link, which is noise for those users. I'd suggest adding these as borders instead.

Reply:
That's why I removed them in the past!